barcode_utils.py

```diff
@@ -0,0 +1,166 @@
+from barcode import *
+
+import base64
+import struct
+import skip32
+import code128
+import os
+
+
```

Comment:
copy that, yea I fixed this in a later pull.  the reason was this code came from another test project I created as a prototype before transplanting that stuff into here.  it's since been massively rewritten